apcd-cms/src/client/src/components/Submitter/Exceptions/ExceptionFormPage.tsx

```diff
@@ -0,0 +1,354 @@
+  const [numberOfExceptionBlocks, setNumberOfExceptionBlocks] =
+    useState<number>(1);
+
+  const handleSubmit = async (
+    values: FormValues,
+    { setSubmitting }: { setSubmitting: (isSubmitting: boolean) => void }
+  ) => {
+    setErrorMessage('');
+    const url = `submissions/exception/api/`;
+    try {
+      const response = await fetchUtil({
+        url,
+        method: `POST`,
+        body: values,
+      });
+      if (response.status == 'success') {
+        setIsSuccess(true);
+      }
+    } catch (error: any) {
+      console.error('Error saving data:', error);
+      console.log(url);
+      if (error.response && error.response.data) {
+        setErrorMessage(
+          error.response.data.message ||
+            'An error occurred while saving the data. Please try again.'
+        );
+      } else {
+        setErrorMessage(
+          'An error occurred while saving the data. Please try again.'
+        );
+      }
+    } finally {
+      setSubmitting(false);
+    }
+  };
+
+  const initialValues: FormValues = {
+    exceptionType: '',
+    exceptions: Array.from({ length: numberOfExceptionBlocks }).map(() => ({
+      businessName: 0,
+      fileType: '',
+      fieldCode: '',
+      expiration_date: '',
+      requested_threshold: 0,
+      required_threshold: 0,
+    })),
+    justification: '',
+    requestorName: '',
+    requestorEmail: '',
+    acceptTerms: false,
+    //expirationDateOther: null,
+  };
+
+  return (
+    <div className={styles.root}>
+      <h1>Exception Request</h1>
+      <hr />
+      <p>
+        This form should be completed and submitted only by entities who are
+        eligible for an exception to certain data submission requirements under
+        H.B. 2090 (87(R)) and associated regulations. Please review the
+        legislation and regulation before submitting this form. Links to both
+        can be found on the {''}
+        <a
+          href="https://sph.uth.edu/research/centers/center-for-health-care-data/texas-all-payor-claims-database/index.htm"
+          target="_blank"
+          rel="noreferrer"
+        >
+          Texas All-Payor Claims Database website.
+        </a>
+      </p>
+      <hr />
+      <h4>Select Exception Type</h4>
+      <p>
+        Please select below if you are requesting an exception for threshold
+        submission requirements or for an general, other type of exception.
+      </p>
+      <Formik
+        initialValues={initialValues}
+        validateOnMount={true}
+        validationSchema={validationSchema}
+        onSubmit={handleSubmit}
+      >
+        {({ values, isSubmitting, setFieldValue, resetForm }) => {
+          useEffect(() => {
+            if (isSuccess) {
+              resetForm();
+            }
+          }, [isSuccess, resetForm]);
+          return (
+            <Form id="threshold-form">
+              <div className={styles.fieldRows}>
+                <FormGroup className="field-wrapper required">
```

Comment:
I think the Select Exception Type should move here?
```suggestion
                <FormGroup className="field-wrapper required">
                <h4>Select Exception Type</h4>
      <p>
        Please select below if you are requesting an exception for threshold
        submission requirements or for an general, other type of exception.
      </p>
```